_posts/2021-05-30-cloudnative-bot.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Using OpenFaas to run a fun twitter bot"
+description: "Using OpenFaas to run a fun twitter bot"
+date: 2021-05-20
+image: /images/2021-05-cnbot/background-twitter.png
+categories:
+ - twitter-bot
+ - use-cases
+ - functions
+author_staff_member: cpanato
+dark_background: true
+
+---
+
+Learn how Openfaas and connector-sdk can be used to run a Twitter bot
+
+## Introduction
+
+Bots are everywhere. Some are useful that can help you in your daily work, and others are collecting information about you to use to make an advertisement, and there are others to be fun and entertaining.
+
+The [@CloudNativeBot](https://twitter.com/CloudNativeBot) was created to bring some fun and also to users to use it to get some knowledge. The bot has some slash commands that can be triggered by posting a message on Twitter.
```

Comment:
was created <- sounds too remote, far away, and disassociated.

Replace it with this:

I created @CloudNativeBot because: ...